saged/small_subsets.py

```diff
@@ -0,0 +1,254 @@
+"""
+This benchmark compares the performance of different models in learning to differentiate between
+healthy and diseased gene expression with equal label distributions in the train and val sets."""
+import argparse
+
+import sklearn.metrics
+import yaml
+
+from saged import utils, datasets, models
+
+
+def subset_to_equal_ratio(train_data: datasets.LabeledDataset,
```

Comment:
This is similar to the function in `saged/keep_ratios.py`, right? Could you combine them into a single function?

Reply:
Right, it's an updated version of the one in saged/keep_ratios.py. When I refactor I'm moving this one to `utils.py` and calling changing both benchmarks to just call the utils version